build/admission/gen-admission-secret.sh

```diff
@@ -0,0 +1,55 @@
+#!/bin/bash
+
+set -e
+
+SERVICE=${SERVICE:-"kubeedge-admission-service"}
+SECRET=${SECRET:-"kubeedge-admission-secret"}
+NAMESPACE=${NAMESPACE:-kubeedge}
+CERTDIR=${CERTDIR:-"/etc/kubeedge/admission"}
+ENABLE_CREATE_SECRET=${ENABLE_CREATE_SECRET:-true}
+
+if [[ ! -x "$(command -v openssl)" ]]; then
+    echo "openssl not found"
+    exit 1
+fi
+
+mkdir -p ${CERTDIR}
+echo "creating certs in dir ${CERTDIR} "
+
+cat <<EOF > ${CERTDIR}/csr.conf
+[req]
+req_extensions = v3_req
+distinguished_name = req_distinguished_name
+[req_distinguished_name]
+[ v3_req ]
+basicConstraints = CA:FALSE
+keyUsage = nonRepudiation, digitalSignature, keyEncipherment
+extendedKeyUsage = serverAuth
+subjectAltName = @alt_names
+[alt_names]
+DNS.1 = ${SERVICE}
+DNS.2 = ${SERVICE}.${NAMESPACE}
+DNS.3 = ${SERVICE}.${NAMESPACE}.svc
+EOF
+
+openssl genrsa -out ${CERTDIR}/ca.key 2048
```

Comment:
can we reuse the certificate generation script here?

Reply:
yes, this a good idea, can we add it to a todo list later? Now use it here is more clear :)

Reply:
why bother to do this manually? instead of generating those certs when the service is started as the earlier version?

Reply:
@chendave  Different users have different security standards, we put a script here for users to choose, if they have their own certificate generator, they can use their own

Reply:
pls ignore my comments, it will only support to run with the image after this PR.

Reply:
@fisherxu can you open an issue and mention that here?

Reply:
sure